pages/linux/bluetoothd.md

```diff
@@ -0,0 +1,24 @@
+# bluetoothd
+
+> Daemon to manage bluetooth devices.
+> More information: <https://manned.org/bluetoothd>.
+
+- Start the daemon:
+
+`bluetoothd`
+
+- Start the daemon, logging to stdout:
+
+`bluetoothd --nodetach`
+
+- Start the daemon with a specific configuration file (defaults to `/etc/bluetooth/main.conf`):
+
+`bluetoothd --configfile {{path/to/file}}`
+
+- Start the daemon with verbose output to stderr:
+
+`bluetoothd --debug`
+
+- Start the daemon with verbose output for a specific file:
```

Comment:
What kind of file is this? A configuration file, or some other kind of file?

Reply:
source file, e.g. `profiles/a2dp.c`
The man page says:
> If file options are present, then only debug prints from that source file are printed. The option can be a pattern containing "*" and "?" characters.
>
> Example: --debug=src/adapter.c:src/agent.c

Reply:
Oh weird.  Then maybe something like this?
```suggestion
- Start the daemon with verbose output coming from specific files in the bluetoothd source:
```